streaming/streaming.go

```diff
@@ -345,8 +346,16 @@ func (c *Conn) processTrade(t TradeUpdate) error {
 		return errors.New("streaming: nonpositive trade")
 	}
 
+	last := c.lastTrade
```

Comment:
I actually don't think rolling back lastTrade is necessary. I would just do:

```
 c.lastTrade = TradeUpdate{Base: t.Base, Counter: t.Counter}
```

here and that's it.

Reply:
Also I don't think we need the order ID. We're dropping one order ID or the other anyway - might as well not include either.

Reply:
Not sure what if streaming process fails? Do you mean if streaming receives an update, this means transaction had already processed, streaming may not process it but it actually is the last trade and should update it any case?

Relating with not setting order ID: makes sense, no need to export it. Will amend it